setup.cfg

```
@@ -88,6 +88,8 @@ dev =
     pyright
     pytest-dependency
     pytest_httpserver
+    # Required for testing snippets in docs.
+    mlflow-skinny
```

Suggestion: can we do it this way so that any automatic sorting won't separate the dependency from the comment?
```suggestion
    mlflow-skinny  # Required for testing snippets in docs.
```

---

OH YES

---

b00477f

---

Nice :D